### DISCUSSION TITLE
How do I check badge advancement (does it include past organizations)

### DISCUSSION THREAD

**Discussion by H-G-11** [community/community#89807] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hey all,

Out of curiosity, I wanted to know if there was a way to check how far I am from the next Pull Request shark badge.

I know that we can check using the API how many requests we did in our organisation.
But as most of us here, I also worked in past organisations, and the number of PR I did there cannot be accessed.

If this is not possible to see, I would love to have this possibility. It helps with motivation and that would be a great feature.

Thanks a lot

**Comment by moritakaaz** [comment]
Hii 👋🏻

![Screenshot_2024-01-19-23-48-18-20_82cebf612ef3f9a8762672c971adaed4.jpg](https://github.com/community/community/assets/67718632/b45f6a76-537c-4a0c-bd7d-d577888a76cb)

Pull Shark seem to be check your PR record and show Level (not your PR count status)
x1 - over 2 PR
x2 - over 16 PR
x3 - over 128 PR
x4 - over 1024 PR

Also "inaccessible" seem to be private PR.
I agree new users might be confused.

**Reply by H-G-11** [reply]
Thank you very much for your answer, I really appreciate.

My question was more the following: how do I know how far I am from the 1024 PR?
It's hard to know if I am at 400 or 800.

Thanks a lot for your time!

**Comment by ghost** [comment]
Hey @H-G-11 So I am using github profile Trophies, so there I could see the number of pull request I had done...

**Reply by ghost** [reply]
![Screenshot_2024-01-19-23-47-17-691_com android chrome](https://github.com/community/community/assets/121353576/c4aa5a85-a331-451a-9fbb-ce85e2d0bef3)


So here you can see the number PR I did is 123

**Reply by H-G-11** [reply]
Ok interesting, thanks for sharing.

I would expect that they use the API from Github, so probably the past organisations would not count then.

I'll still try it out, thanks and have a great day

**Reply by ghost** [reply]
Your Welcome
